terraform-modules/aws/secret-manager/README.md

```diff
@@ -0,0 +1,80 @@
+# AWS Secret Manager
+
+Certainly! This Terraform configuration creates an empty AWS Secrets Manager secret. 
```

Comment:
Why does this paragraph start with "Certainly!"?

Reply:
Since you are in here editing this...add in the "why"

Why you would want to use this module?
* To be able to create the AWS Secret resource via IAC but be able to set the actual secret value by ClickOps.

Reply:
added comments and removed word **Certainly**